command/agent/check_test.go

```diff
@@ -65,19 +71,38 @@ func expectStatus(t *testing.T, script, status string) {
 }
 
 func TestCheckMonitor_Passing(t *testing.T) {
-	expectStatus(t, "exit 0", structs.HealthPassing)
+	expectStatus(t, "exit 0", "", structs.HealthPassing)
+}
+
+func TestCheckMonitor_Chroot(t *testing.T) {
+	// running this test only on linux and if the uid is root
+	if runtime.GOOS != "linux" || syscall.Getuid() != 0 {
```

Comment:
@sean- Yeah, the actual implementation would work on any platform but windows. However, I restricted the test code to run only on linux since I was copying the dependencies ```bash``` needed.